packages/web-components/fast-foundation/docs/integrations/aurelia.md

```diff
@@ -101,89 +88,125 @@ fast-card > fast-button {
+npm install aurelia-fast-adapter
+```
 
-export class AureliaFastAdapter {
+and then simply register it from your `src/main.ts`:
 
-  public static register(container: IContainer) {
-    AureliaFastAdapter.extendTemplatingSyntax(container);
-  }
+```ts
+// src/main.ts
 
-  private static extendTemplatingSyntax(container) {
-    AppTask.with(IContainer).beforeCreate().call(container => {
-      const attrSyntaxTransformer = container.get(IAttrSyntaxTransformer);
-      const nodeObserverLocator = container.get(NodeObserverLocator);
-      attrSyntaxTransformer.useTwoWay((el, property) => {
-        switch (el.tagName) {
-          case 'FAST-SLIDER':
-          case 'FAST-TEXT-FIELD':
-          case 'FAST-TEXT-AREA':
-            return property === 'value';
-          case 'FAST-CHECKBOX':
-          case 'FAST-RADIO':
-          case 'FAST-RADIO-GROUP':
-          case 'FAST-SWITCH':
-            return property === 'checked';
-          case 'FAST-TABS':
-            return property === 'activeid';
-          default:
-            return false;
-        }
-      });
-      // Teach Aurelia what events to use to observe properties of elements.
-      const valuePropertyConfig = { events: ['input', 'change'] };
-      nodeObserverLocator.useConfig({
-        'FAST-CHECKBOX': {
-          checked: valuePropertyConfig
-        },
-        'FAST-RADIO': {
-          checked: valuePropertyConfig
-        },
-        'FAST-RADIO-GROUP': {
-          value: valuePropertyConfig
-        },
-        'FAST-SLIDER': {
-          value: valuePropertyConfig
-        },
-        'FAST-SWITCH': {
-          checked: valuePropertyConfig
-        },
-        'FAST-TABS': {
-          activeid: valuePropertyConfig
-        },
-        'FAST-TEXT-FIELD': {
-          value: valuePropertyConfig
-        },
-        'FAST-TEXT-AREA': {
-          value: valuePropertyConfig
-        }
-      });
-    }).register(container);
-  }
-}
+import { FASTAdapter } from './aurelia-fast-adapter';
+
+Aurelia
+  .register(FASTAdapter) // add this line
+  // other registrations...
+  .start();
 ```
 
-Then, open your `src/main.ts` file and register your new adapter as such: 
+If you use FAST in its default configuration that's all you need to do. But if you changed the prefix of your components to something else, you can customize the adapter as such:
 
 ```ts
-// main.ts
+// src/main.ts
 
-import { AureliaFastAdapter } from './aurelia-fast-adapter';
+import { FASTAdapter } from './aurelia-fast-adapter';
 
 Aurelia
-  .register(AureliaFastAdapter) // add this line
-  // other registrations...
+  .register(FASTAdapter.customize({withPrefix: 'my-custom-prefix'}) // customized with prefix
+  .start();
+```
+
+Also, in case you have local components that also require two-way binding, you can adjust the adapter before to register it as such:
+
+```ts
+// src/main.ts
+
+import { FASTAdapter } from './aurelia-fast-adapter';
```

Comment:
```suggestion
import { FASTAdapter } from 'aurelia-fast-adapter';
```

`aurelia-fast-adapter` is a package right? not a file?

Reply:
good catch thanks. This is now fixed